### DISCUSSION TITLE
Sort sounds

### DISCUSSION THREAD

**Discussion by Noctabou** [Tom4nt/Mega-Soundboard#45] [Ideas]
Hello Tom4nt,
I've been using the app for a long time, and i love it. 
I have one idea to add to the soundboard : A button to sort the sounds by alphabetical order.
It could be placed next to the Add sound button.
I already have a trick to sort sounds ; Delete and add my folder back in MegaSoundboard. It works ! But it will reset all my volumes :/

**Comment by Tom4nt** [comment]
Thanks for the suggestion! Added to the planned features.
